src/i18n/es/index.ts

```diff
@@ -78,6 +80,8 @@ const es = {
 	run: 'Ejecutar',
 	save: 'Guardar',
 	saveAndRun: 'Guardar y ejecutar',
+	search: 'Buscar',
+	searchEmpty: 'No se encontraron resultados',
```

Comment:
Were these missing from my PR?

I think we need to add a CI check for this i18n stuff, it's easy to forget to translate stuff and/or re-generate the types.